src/dltParser.ts

```diff
@@ -292,7 +295,7 @@ export class DltMsg {
                                     argOffset += 2;
                                     const rawd = this._payloadData.slice(argOffset, argOffset + lenRaw);
                                     this._payloadArgs.push({ type: Buffer, v: Buffer.from(rawd) }); // we make a copy here to avoid referencing the payloadData that we want to release/gc afterwards
-                                    this._payloadText += rawd.toString("hex");
+                                    this._payloadText += rawd.toString("hex") + this.payloadArgSeparator;
```

Comment:
I think it's not the only place where the space is missing?
And: does dltviewer add a space at the end?

Reply:
Hard to tell.
Do you have an easy to use DLT example that uses all different DLT_LOG functions to write messages? Plus, I'm not really that good at reading the dltviewer code to answer that clearly.

What I would say: the current implementation, or rather, simply adding a space at this position did not seem to add additional spaces at the end in vscode.

Reply:
Sadly I miss good examples. Could write a small test app that creates those.
I might have some code to remove whitespace at end ;-) (will take a look)
I can check the dltviewer code and try to understand the rules implemented there.